apps/src/publicKeyCryptography/main.js

```diff
@@ -3,16 +3,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import PublicKeyCryptographyWidget from './PublicKeyCryptographyWidget';
+import ModuloClockWidget from './ModuloClockWidget';
 
-function initialize() {
+// These constants should stay in sync with the levelbuilder options on
+// _public_key_cryptography.html.haml
+const MODULO_CLOCK_VIEW = 1;
+const ALICE_EVE_BOB_VIEW = 2;
+
+function initialize(options) {
   ReactDOM.render(
-      <PublicKeyCryptographyWidget/>,
+      parseInt(options.cryptographyWidgetView, 10) === ALICE_EVE_BOB_VIEW ?
+          <PublicKeyCryptographyWidget/> :
+          <ModuloClockWidget/>,
       document.getElementById('public-key-cryptography-mount'));
 }
 
 // Start initialization when DOM is ready.
 function onDOMContentLoaded() {
   document.removeEventListener('DOMContentLoaded', onDOMContentLoaded, false);
-  initialize();
+  // Get options / default options
+  const options = {
+    cryptographyWidgetView: MODULO_CLOCK_VIEW,
+    ...window.options
```

Comment:
guess we did add "transform-object-rest-spread", to .bablerc at some point, so should be 👍